content/en/profiler/profile_types.md

```diff
@@ -158,6 +158,18 @@ CPU
 Wall Time
 : The elapsed time used by each function. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the function is running.
 
+Allocations (alpha, v1.19.0+)
+: The number of objects allocated by each method during the profiling period (default: 60s), including allocations which were subsequently freed. This is useful for investigating garbage collection load.<br />
+_Requires: Ruby 2.7+_
```

Comment:
The framing before the list suggests these are on by default. Shall we mention here the env variables that are also required or something along the lines of `Requires: Ruby 2.7+ and [manual enablement](link to release notes)`

Reply:
That's a good point. One thing I spotted is that we seem to make the same mistake everywhere -- e.g. for the other languages we don't have enablement info and we make it seem like all features show up by default.

Which means I guess we get to be the first ones? ;) 

I slightly hesitate before linking to the release notes, though. I guess https://docs.datadoghq.com/profiler/enabling/ruby would be a nicer spot to link to?

Reply:
Mentioning alpha features on what looks/works essentially as a quickstart page feels dangerous 😅 

Didn't we have an "Advanced" section somewhere in our docs before?

Reply:
Instructions for specific configurations might be appropriate for a new guide: https://docs.datadoghq.com/profiler/guide/

Reply:
Ok, I propose that for now we go with the quick link off to the release notes; and once we go beta we'll give it a second pass and add it to the docs.

I'll wait until the actual release is out to have the correct link/anchor to the release notes to add here.

Reply:
> Instructions for specific configurations might be appropriate for a new guide: [docs.datadoghq.com/profiler/guide](https://docs.datadoghq.com/profiler/guide/)

Not really sure that would be a good approach. Based on the existing guide, it seems we're after a more learning-focused "tutorial" format rather than "here's how to turn it on". Left unchecked we could end up with so many random guides that it'd hide the real gems.

OTOH, it'd actually be great to have a more general "how to profile memory" tutorial with subsections for how to enable it on different languages. It's just that it'd be a lot of work and a major stepping stone in the way of just announcing a new alpha feature 😅